1-mandatory/2-quotegenerator/quotes.js

```diff
@@ -490,3 +502,34 @@ const quotes = [
     author: "Zig Ziglar",
   },
 ];
+// pseudo Code
+// 1. display the first quote from the Array.
+// 2. pick a random quote from array using Math.random
+// 3. display a random quote from array when a screen first load
+// 4.when we click a button in screen,it should change a quote.
+
+const quotesHolder = document.getElementById("quote-holder");
+const quotePara = document.getElementById("quote");
+const quoteAuthor = document.getElementById("author");
+
+function getRandomQuote(){
+  const randomNumber = Math.floor(Math.random()* quotes.length);
+  console.log(randomNumber);
+  quotePara.textContent = quotes[randomNumber].quote;
+  quoteAuthor.textContent = quotes[randomNumber].author;
+}
+getRandomQuote();
+
+const button = document.getElementById("new-quote");
+button.addEventListener("click", function(){
+  getRandomQuote();
+});
+
+// function generateNewQuotes (){
+// const newQuote = pickFromArray(quotes)
+// headingEle.innerText=newQuote["quote"]
+// let newAuthor = pickFromArray(quotes)
+// pEle.innerText = newAuthor["author"]
+
+// };
+// buttonEle.addEventListener("click", generateNewQuotes)
```

Comment:
Nicely done, and I like the pseudocode! Only thing I would say is to keep your workspace clean - remember to remove commented out code (a.k.a. "zombie code") and console logs when you're done with them.